src/openrct2-ui/windows/About.cpp

```diff
@@ -59,15 +58,17 @@ namespace OpenRCT2::Ui::Windows
     };
 
     static constexpr auto kMainWidgets = makeWidgets(
-        makeWindowShim(WINDOW_TITLE, { WW, WH }),
-        makeWidget({ 0, TABHEIGHT }, { WW, WH - TABHEIGHT }, WidgetType::frame, WindowColour::secondary),
-        makeRemapWidget({ 3, 17 }, { 91, TABHEIGHT - 16 }, WidgetType::tab, WindowColour::secondary, SPR_TAB_LARGE),
-        makeRemapWidget({ 94, 17 }, { 91, TABHEIGHT - 16 }, WidgetType::tab, WindowColour::secondary, SPR_TAB_LARGE));
+        makeWindowShim(kWindowTitle, kWindowSize),
+        makeWidget(
+            { 0, kTabHeight }, { kWindowSize.width, kWindowSize.height - kTabHeight }, WidgetType::frame,
+            WindowColour::secondary),
```

Comment:
Can we extend `// clang-format off` (just below) to this block to keep this on one line?